jlh/JonFirstApp/app/src/main/java/com/e_conomic/jonfirstapp/MainActivity.java

```diff
@@ -1,13 +1,33 @@
 package com.e_conomic.jonfirstapp;
 
+import android.content.Intent;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.EditText;
 
 public class MainActivity extends AppCompatActivity {
 
+    // Key for the intent extra holding the message.
+    public final static String EXTRA_MESSAGE = "com.e-conomic.jonfirstapp.MESSAGE";
```

Comment:
Haha. A bit overkill here obviously, but the idea presented in the tutorial was that you should make sure the keys are still unique if the app interacts with another app. Do you guys have a convention for such keys (or what do you usually do)? :)